common/src/main/java/org/dash/wallet/common/data/entity/ExchangeRate.kt

```diff
@@ -34,16 +34,25 @@ open class ExchangeRate : Parcelable {
     @PrimaryKey
     var currencyCode: String
     var rate: String? = null
+    @Ignore
+    var retrievalTime: Long = -1L
 
     @Ignore
     private var currencyName: String? = null
 
     @Ignore
     private var currency: Currency? = null
 
+    constructor(currencyCode: String, rate: String?, retrievalTime: Long) {
+        this.currencyCode = currencyCode
+        this.rate = rate
+        this.retrievalTime = retrievalTime
```

Comment:
CTX does return a field that contains the time their servers last retrieved this particular rate.

It is possible for CTX to return rates, but they are stale.